wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,75 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+A user may get banned if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What are considered cheating?
```

Comment:
"what is considered cheating"

link to the rules wiki page, maybe even cite it directly in a blockquote